frontend/components/modals/ExecuteWithOtherWalletModal.vue

```diff
@@ -1,16 +1,12 @@
 <template>
     <div>
-        <div class="flex flex-row-reverse md:mb-0">
-            <base-button
-                :disabled="disabled"
-                class="w-full md:w-80"
-                :is-loading="isLoading"
-                @click="setModalVisibility(true)"
-            >
-                Execute to another wallet
-            </base-button>
-        </div>
-        <modal v-model="isVisible" title="Execute to another wallet" destroy-on-close>
+        <modal :visible="isShown" title="Execute to another wallet" destroy-on-close @cancel="$emit('close')">
+            <Alert v-if="!isOutcomeWalletInputValueValid && outcomeWalletInputValue" type="error" banner>
+                <template slot="message">
+                    Address <span class="font-bold">{{ outcomeWalletInputValue }}</span> is not a valid address.
+                </template>
+            </Alert>
```

Comment:
Please move the error closer to the source of error, right above or below the input. Not sure why did you decide to move it up?